### PR TITLE
feat: update tab name with update timestamp

### PR DESCRIPTION
**Pull request by rephus**
Closes: #6671

### Description:

image

[Screencast from 14-08-23 16:14:34.webm](https://github.com/lightdash/lightdash/assets/1983672/feba2638-2581-4d6c-9968-d4e7caa37aea)

### CODE REVIEW

**Review by hamzahc1 — APPROVED** [review]
This looks good to me! (I reviewed via the video)

File: packages/backend/src/clients/Google/GoogleDriveClient.ts [hunk 1]
```diff
@@ -50,6 +50,34 @@ export class GoogleDriveClient {
         });
     }
 
+    private static async changeTabTitle(
+        sheets: any,
```
Comment (joaoviana) [inline packages/backend/src/clients/Google/GoogleDriveClient.ts:54]: Maybe this was answered in previous PRs, but was wondering why we're not referencing Google's types from `googleapis` - so this one would be:
```ts
sheets: sheets_v4.Sheets
```

I had to update the import to `import { google, sheets_v4 } from 'googleapis';` so I could get typescript to show me the errors btw 👍

Comment (rephus) [reply packages/backend/src/clients/Google/GoogleDriveClient.ts:54]: Yes of course, great suggestion, I'll fix the types , but perhaps not in this PR but on the main PR or maybe another PR.

Comment (joaoviana) [reply packages/backend/src/clients/Google/GoogleDriveClient.ts:54]: sounds good! thanks 👍

**Pull request merged**